openshift_cli_installer/libs/clusters/ocp_clusters.py

```diff
@@ -46,7 +48,10 @@ def __init__(self, **kwargs):
     def add_to_cluster_lists(self, ocp_cluster, **kwargs):
         _cluster_platform = ocp_cluster["platform"]
         if _cluster_platform == AWS_STR:
-            self.aws_ipi_clusters.append(AwsIpiCluster(ocp_cluster=ocp_cluster, **kwargs))
+            self.aws_ipi_clusters.append(IpiCluster(ocp_cluster=ocp_cluster, **kwargs))
```

Comment:
We talked about creating a class for each, `AwsIpiCluster` and `GcpIpiCluster`

Reply:
yes, working on it